### PR TITLE
Fix pyinstaller example

### PR DESCRIPTION
**Pull request by colbysparks**
Add bash build script and documentation for linux/macOS users.
Fix build.bat script for windows.
Update zml version to normalize go lib load path.

### CODE REVIEW

**Comment by colbysparks** [comment]
Going to wait to bump zml version when we release the lib load path fix.

File: examples/util_pyinstaller/README.md [hunk 1]
```diff
@@ -53,3 +53,20 @@ You can test it by running:
...
+
+## Building the Executable (Linux/macOS)
+
+```shell
+cd examples/util_pyinstaller/
+pdm install
+./build.sh
+```
+
+Note to linux users that you may need to use `sudo`
```
Comment (martinzak-zaber) [inline examples/util_pyinstaller/README.md:65]:
```suggestion
Note to Linux users that you may need to use `sudo`.
```

File: examples/util_pyinstaller/build.bat [hunk 2]
```diff
@@ -1,2 +1,2 @@
 call .venv\Scripts\activate.bat
-pyinstaller --onefile --add-binary ".venv\Lib\site-packages\zaber_motion_bindings\zaber-motion-core-windows-amd64.dll;zaber_motion_bindings" main.py
+pyinstaller --onefile --add-binary ".venv\Lib\site-packages\zaber_motion_bindings\zaber-motion-core-windows-amd64.dll;zaber_motion_bindings" src/util_pyinstaller/main.py
```
Comment (martinzak-zaber) [inline examples/util_pyinstaller/build.bat:2]:
```suggestion
pyinstaller --onefile --add-binary ".venv\Lib\site-packages\zaber_motion_bindings;zaber_motion_bindings" src/util_pyinstaller/main.py
```

also seem to work.

File: examples/util_pyinstaller/pyproject.toml [hunk 3]
```diff
@@ -6,8 +6,8 @@ authors = [
     {name = "Martin Zak", email = "software@zaber.com"},
 ]
 dependencies = [
-    "zaber-motion==6.4.0",
     "pyinstaller==6.5.0",
+    "zaber-motion>=7.2.1",
```
Comment (martinzak-zaber) [inline examples/util_pyinstaller/pyproject.toml:10]:
```suggestion
    "zaber-motion==7.2.3",
```
to be sure.

File: examples/util_pyinstaller/build.bat (hunk 2 above)
Comment (colbysparks) [reply examples/util_pyinstaller/build.bat:2]: On my linux VM:
```
pyinstaller: error: argument --add-binary: Wrong syntax, should be --add-binary=SOURCE:DEST
```

File: examples/util_pyinstaller/README.md (hunk 1 above)
Comment (colbysparks) [reply examples/util_pyinstaller/README.md]: I've just removed this line. Linux users will know what to do

**Review by martinzak-zaber — APPROVED** [review]

**Pull request merged**